.jvmopts

```diff
@@ -1 +1,5 @@
 -Dcats.effect.stackTracingMode=full
+-Dfile.encoding=UTF8
+-Xms265M
+-Xmx12G
+-XX:+UseG1GC
```

Comment:
Let's revert this one. You can adjust the `.sbtopts` file which is not git tracked and set adequate values depending on your platform / machine